Documentation/building/testing-with-corefx.md

```diff
@@ -3,134 +3,144 @@ Testing with CoreFX
 
 It may be valuable to use CoreFX tests to validate your changes to CoreCLR or mscorlib.
 
-## Building CoreFX against CoreCLR
+# Building CoreFX against CoreCLR
 **NOTE:** The `BUILDTOOLS_OVERRIDE_RUNTIME` property no longer works.
 
 To run CoreFX tests with an updated System.Private.Corelib.dll, [use these instructions](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits).
 
-To build CoreFX against the updated System.Private.Corelib.dll - we need to update instructions.
+To build CoreFX against the updated System.Private.Corelib.dll - _we need to update instructions_.
```

Comment:
The instruction from https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bit on the previous line cover both building the CoreFX product bits and running the tests. We can drop this line.